components/announcement_bar/configuration_bar/configuration_bar.tsx

```diff
@@ -33,6 +35,8 @@ type Props = {
     user?: UserProfile;
     canViewSystemErrors: boolean;
     totalUsers?: number;
+    dismissedExpiringTrialLicenseLastDay?: boolean;
+    dismissedExpiringTrialLicense?: boolean;
```

Comment:
I don't understand the need for `dismissedExpiringTrialLicenseLastDay` variable. Seems like one is all that is necessary.

Reply:
Ya you're right. I goofed. We don't need it.